setup.py

```diff
@@ -64,6 +64,25 @@ def finalize_options(self):
         install.finalize_options(self)
 
 
+cmdClass = {'install': CheckCairoXcb}
+
+try:
+    from sphinx.setup_command import BuildDoc
+    import subprocess
+
+    class BuildDocWrapper(BuildDoc):
+        def run(self):
+            subprocess.call(['scripts/ffibuild'])
```

Comment:
Why do we need this? I guess that the genkeyimg task needs `scripts/ffibuild` to have been executed? If so, I'd rather make this as another task in the Makefile and depend on it there, no?

Reply:
I don't remember why I chouse that path, but we need working `configreader`. As it doesn't change logics, I'd like faster to deploy and then later refactor if we will see that we need `ffbuild` somewhere else

Reply:
sounds fair